<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Expence tracker</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(81, 0, 135, 255), stop:0.427447 rgba(41, 61, 132, 235), stop:1 rgba(155, 79, 165, 255));</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>12</number>
     </property>
     <property name="topMargin">
      <number>12</number>
     </property>
     <property name="rightMargin">
      <number>12</number>
     </property>
     <property name="bottomMargin">
      <number>12</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QFrame" name="balance_frame">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Current Balance</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>$0</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="resourse.qrc">:/resourse/icon/north_west_white_24dp.svg</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Income</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>$0</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="resourse.qrc">:/resourse/icon/call_received_white_24dp.svg</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Outcome</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>$0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="categories_frame">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>12</number>
          </property>
          <property name="topMargin">
           <number>12</number>
          </property>
          <property name="rightMargin">
           <number>12</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Expensies Categories</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_10">
                <property name="maximumSize">
                 <size>
                  <width>24</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="resourse.qrc">:/resourse/icon/local_grocery_store_white_24dp.svg</pixmap>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Groceries</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>-0$</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_14">
                <property name="maximumSize">
                 <size>
                  <width>24</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="resourse.qrc">:/resourse/icon/sports_esports_white_24dp.svg</pixmap>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Entertainment</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>-0$</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_16">
                <property name="maximumSize">
                 <size>
                  <width>24</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="resourse.qrc">:/resourse/icon/directions_car_white_24dp.svg</pixmap>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Auto</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>-0$</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_19">
                <property name="maximumSize">
                 <size>
                  <width>24</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="resourse.qrc">:/resourse/icon/list_white_24dp.svg</pixmap>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>Other</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>-0$</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QPushButton" name="pushButton_1">
         <property name="text">
          <string>New Transaction</string>
         </property>
         <property name="icon">
          <iconset resource="resourse.qrc">
           <normaloff>:/resourse/icon/post_add_white_24dp.svg</normaloff>:/resourse/icon/post_add_white_24dp.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Delete Transaction</string>
         </property>
         <property name="icon">
          <iconset resource="resourse.qrc">
           <normaloff>:/resourse/icon/delete_white_24dp.svg</normaloff>:/resourse/icon/delete_white_24dp.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>New month</string>
         </property>
         <property name="icon">
          <iconset resource="resourse.qrc">
           <normaloff>:/resourse/icon/edit_white_24dp.svg</normaloff>:/resourse/icon/edit_white_24dp.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>135</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resourse.qrc"/>
 </resources>
 <connections/>
</ui>
